src/bin/resource_tracing.rs

```diff
@@ -1284,6 +1367,106 @@ fn main()
         }
         println!("            ],");
     }
+    println!("                    // TODO: these are just copies of nothing - the actual data is handled during patching
+    PickupType::UnknownItem1 => &[
```

Comment:
Instead of hard coding the byte sequence here, can we actually print nothing's real contents? Alternatively, if this data is actually never read, can we just print an empty slice? Or hell, if this shouldn't be used without completely custom data from the user, then print a panic? Same with the resource dep-list above